source/read/change-streams.txt

```diff
@@ -147,8 +147,8 @@ of the watch functions:
      - Description
 
    * - ``batchSize``
-     - | Sets the number of documents to return per batch.
-
+     - | The maximum number of documents within each batch returned 
+         in a change stream, which applies to the watch function. By default, watch functions have an initial batch size of ``101`` documents and a maximum size of 16 mebibytes (MiB) for each subsequent batch. This option can enforce a smaller limit than 16 MiB, but not a larger one. If you set ``batchSize``  to a limit that results in batches larger than 16 MiB, this option has no effect and the watch function uses the default batch size.
```

Comment:
S: line 151 looks pretty long, so I'd break it up over a few lines. Also I think this wording for the first sentence is a little more clear / uses active voice:
```suggestion
     - | The maximum number of documents that a change stream can
         return in each batch, which applies to the watch function. By default, watch functions have an initial batch size of ``101`` documents and a maximum size of 16 mebibytes (MiB) for each subsequent batch. This option can enforce a smaller limit than 16 MiB, but not a larger one. If you set ``batchSize``  to a limit that results in batches larger than 16 MiB, this option has no effect and the watch function uses the default batch size.
```